.storybook/stories/tabs/tabs.stories.ts

```diff
@@ -56,20 +43,38 @@ const defaultParameters: Parameters = {
   },
 };
 
-setupStorybook(ClrTabsModule, defaultStory, defaultParameters, generateVariants());
+const tabsTemplate: Story = args => ({
+  template: `
+    <clr-tabs [clrLayout]="clrLayout">
+      <clr-tab *ngFor="let _ of createArray(tabCount); let i = index">
+        <button clrTabLink>{{title}} {{i + 1}}</button>
+        <clr-tab-content *clrIfActive="activeTab === (i + 1)">
+          <p>
+            {{content}} {{i + 1}}
+          </p>
+        </clr-tab-content>
+      </clr-tab>
+    </clr-tabs>
+  `,
+  props: { ...args },
+});
 
-function generateVariants() {
-  const variants: Parameters[] = [];
+export const Tabs: StoryObj = {
+  render: tabsTemplate,
+};
 
-  for (const clrLayout of [TabsLayout.HORIZONTAL, TabsLayout.VERTICAL]) {
-    for (const activeTab of [1, 2, 3, 4]) {
-      variants.push({
-        clrLayout,
-        activeTab,
-        tabCount: 4,
-      });
-    }
-  }
+export const VerticalTabs: StoryObj = {
+  render: tabsTemplate,
+  args: {
+    clrLayout: TabsLayout.VERTICAL,
+  },
+};
 
-  return variants;
-}
+export const TabsReactive: StoryObj = {
+  render: tabsTemplate,
+  parameters: {
+    viewport: {
+      defaultViewport: 'large',
+    },
+  },
+};
```

Comment:
What does this add? The screenshot looks the same to me.

Reply:
There is another section called 'Tabs Reactive'. After speaking with @mivaylo I see this is to demo 'responsiveness' if I understand correctly.

Reply:
I do want to understand though if this is required, seems unnecessary. For showcasing, there is horizontal and vertical tabs for variants.

Reply:
Okay, let's keep it as suggested in that case.

Reply:
I think we should use the term "responsive" instead of "reactive" for this.